test/locale.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { classes } from "../src/commands";
+import { COMMAND_LOCALIZATIONS, loadTranslations } from "../src/locale";
+
+describe(".po files", () => {
+	test("syntax is valid", () => {
+		const locales = loadTranslations();
```

Comment:
Is there a way to load them once in the "set up" or is there a particular reason we want to reload them for each test?

Reply:
The loading is actually the test. If the syntax is invalid, this will fail.